yaksh/test_models.py

```diff
@@ -384,6 +384,26 @@ def setUpClass(self):
         self.questions = Question.objects.filter(active=True)
         self.quiz = Quiz.objects.get(description="demo quiz 1")
 
+        # create question paper with only fixed questions
+        self.question_paper_fixed_questions = QuestionPaper.objects.create(
+                quiz=self.quiz)
+        self.question_paper_fixed_questions.fixed_questions.add(
+                self.questions.get(id=11), self.questions.get(id=10))
```

Comment:
It will be better if you fetch question with some other parameter instead of id. This id might change at some point.

Reply:
Creating specific questions for this particular test case would be a good idea.

Reply:
Okay then used summary instead of id.
@ankitjavalkar Actually I only need to show that the required questions are retrieved.
So questions can be any from the present set. Then too any suggestion then please mention.